### ISSUE TITLE
`UNIQUE` index is not checked in presence of PK.

### ISSUE THREAD

**Issue by kyukhin** [tarantool/tarantool#2255]
This snippet must fail:
```
box.sql.execute("CREATE TABLE t1(a integer primary key, b);");                                                                                                                                                                                                                  
box.sql.execute("INSERT INTO t1 VALUES(1,3);");                                                                                                                                                                                                                                 
box.sql.execute("INSERT INTO t1 VALUES(2,3);");                                                                                                                                                                                                                                 
                                                                                                                                                                                                                                                                                
box.sql.execute("CREATE UNIQUE INDEX t2b ON t1(b);");       
```
since second column (on which index is created) contains duplicates.
The error wasn't triggered since right now, when creating index in Tarantool
we append [unique] PK value, so uniqueness is respected.

**Comment by kyukhin** [comment]
Blocked by #1557 

The goal is to remove uniqueness check from SQL VM.
This is responsibility of Tarantool's indexes.

**Comment by Khatskevich** [comment]
After resolving #1557 a5ce84855a5936578a258e7f45674b7e04678051 should close this issue.

**Comment by kyukhin** [comment]
Sounds, like this is duplicate of #2495
